packages/tables/src/Concerns/CanSortRecords.php

```diff
@@ -113,6 +113,10 @@ protected function applyDefaultSortingToTableQuery(Builder $query): Builder
             return $query->orderBy($sortColumnName, $sortDirection);
         }
 
+        if (filled($query->toBase()->orders)) {
```

Comment:
I think this clock needs to be moved to line 128, before we apply the ID sort? Otherwise, `defaultSort()` with a custom query function doesn't work with the global scope

Reply:
moved it